packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -44,5 +44,7 @@
     "SFDX: Execute Anonymous Apex with Currently Selected Text",
   "feature_previews_title": "Salesforce Feature Previews",
   "force_project_create_text": "SFDX: Create Project",
-  "force_apex_trigger_create_text": "SFDX: Create Apex Trigger"
+  "force_apex_trigger_create_text": "SFDX: Create Apex Trigger",
+  "show_cli_success_msg_description":
+    "Specifies whether the SFDX CLI messages executed via vscode will show as information messages (true) or as status bar messages (false)."
```

Comment:
@ruthemmanuelle this text will be part of the default settings file users see above the newly added setting option (salesforcedx-vscode-core.show-cli-success-msg).

Reply:
How about this?
"Specifies whether status messages for Salesforce CLI commands run using the VS Code command palette will appear as pop-up information messages (true) or as status bar messages (false)."

Reply:
Cool, I'll add that change before merging.